domains/certificates/Query.API/API/MeasurementsSyncer/MeasurementsSyncService.cs

```diff
@@ -3,67 +3,58 @@
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
+using API.Configurations;
 using API.MeasurementsSyncer.Metrics;
 using API.MeasurementsSyncer.Persistence;
 using DataContext.Models;
 using DataContext.ValueObjects;
 using Measurements.V1;
 using Meteringpoint.V1;
 using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
 
 namespace API.MeasurementsSyncer;
 
-public class MeasurementsSyncService
+public class MeasurementsSyncService(
+    ILogger<MeasurementsSyncService> logger,
```

Comment:
What happened to the underscores?

Reply:
Primary constructors happened

Reply:
I see that :) But why did the wish for a primary constructor suddenly overshadow last weeks unanimous decision on using _ in fields?